docs/Firewall-Automation.md

```diff
@@ -41,86 +130,63 @@ _The Azure CLI Automation extension is in an experimental stage.  Currently it d
 
-### Resources Created When Complete
+## Resources Created When Complete
 
-1. Azure user-assigned Managed Identity
+1. Azure User-Assigned Managed Identity
 2. Azure Automation Account
 3. Azure PowerShell Runbook
 
     ![List of resources that will be created when complete.](./assets/Firewall-Automation/listOfResourcesInResourceGroup.png)
 
-## Installation Method - Automated Scripts
+## Runbook and Schedule Creation using Automated Scripts
 
-BEFORE continuing please make sure all requirements have been met in the section labeled [prerequisites]("#-prerequisites").
+The steps to set up the runbook and schedule are listed in this section. BEFORE continuing please make sure all requirements have been met in the section labeled [prerequisites](#prerequisites).
 
-1. [Create Automation Infrastructure (BASH script)]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure.sh")
+### Execute Script
 
-    The [Firewall-Automation-Infrastructure.sh]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure.sh") script "dot sources" the [Firewall-Automation-Infrastructure-Variables.sh]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure-Variables.sh").  
-
-_This file must to be adjusted for the specifics of the execution._
-
-### BASH Variables
-
-The BASH variables are exported into environment variables.  
+Once the variables are updated, the setup script must be run from Visual Studio Code (thick client) using Codespaces. The script does not require input parameters because the required parameters are stored as environment variables when it runs the variable script. Run this command from the top-level directory of this repository.
 
 ```bash
-# Tenant Id for the onmicrosoft.com tenant
-export ASB_FW_Tenant_Id=''
-export ASB_FW_Subscription_Name=''
-export ASB_FW_Base_NSGA_Name='ngsa-asb'
-export ASB_FW_Base_Automation_System_Name='firewall-automation'
-export ASB_FW_Environment='dev'
-export ASB_FW_PowerShell_Runbook_File_Name='Firewall-Automation-Runbook.ps1'
-export ASB_FW_Sku='Basic'
-export ASB_FW_Location='westus'
-export ASB_FW_PowerShell_Runbook_Description='This runbook allocates and de-allocates specific firewalls.  It also enables and disables specific metric and log alerts associated with such activities.'
-
+  ./scripts/Firewall-Automation/Firewall-Automation-Infrastructure.sh
 ```
 
-Below will detail what is being executed within the script files for further understanding.  This section is informational only.
-
-#### Adjust Variables
-
-The file [Firewall-Automation-Infrastructure-Variables.sh]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure-Variables.sh") must be adjusted to include relevant values.
-
-_Potentially sensitive values such as subscription Id have been omitted from the documentation_
+## Delete Service Principal and Role Assignments
```

Comment:
I recommend adding a bit of context here. Should the user execute these commands after running the script, or is it simply for information on how to clean up? I initially assumed I was supposed to run these commands as well, but would the automation require these permissions to function?

Reply:
done